armi/operators/tests/test_operators.py

```diff
@@ -107,6 +109,25 @@ def test_couplingIsActive(self):
         self.o.cs["tightCoupling"] = True
         self.assertTrue(self.o.couplingIsActive())
 
+    def test_dbWriteForCoupling(self):
+        with directoryChangers.TemporaryDirectoryChanger():
+            self.o.cs["tightCoupling"] = True
+            self.dbWriteForCoupling(writeDB=True)
+            self.dbWriteForCoupling(writeDB=False)
```

Comment:
There are no `assert` statements here.  We should add one.

Reply:
Ah. The assertion lives in the `dbWriteForCoupling` method.